packages/client/src/pages/LoginPage.tsx

```diff
@@ -238,10 +238,12 @@ interface TypeUserRegister {
  * LoginPage
  */
 export const LoginPage = observer(() => {
-    const { configStore } = useRootStore();
+    const { configStore, monolithStore } = useRootStore();
+    const location = useLocation();
 
     const [forgotPassword, setForgotPassword] = useState(false);
-    const [loginType, setLoginType] = useState('Native');
+    const [availableLogins, setAvailableLogins] = useState<string[]>([]);
```

Comment:
You could just leave it in the `loginsAllowed` object and check if they key exists